src/System.IO.FileSystem/src/System/IO/CharSpanExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.IO
+{
+    internal static partial class CharSpanExtensions
+    {
+        public static bool EndsWithOrdinal(this ReadOnlySpan<char> span, ReadOnlySpan<char> value, bool ignoreCase = false)
+        {
+            if (value.Length == 0)
+                return true;
+            else if (span.Length == 0 || value.Length > span.Length)
```

Comment:
Do we need the first part of the condition here?  Couldn't it just be:
```C#
else if (value.Length > span.Length)
```
?

Reply:
Good catch, thanks